gym_ignition/__init__.py

```diff
@@ -79,7 +79,8 @@
     kwargs={'task_cls': cartpole_discrete.CartPoleDiscrete,
             'robot_cls': sim.gazebo.cartpole.CartPoleGazeboRobot,
             'model': "CartPole/CartPole.urdf",
-            'world': "DefaultEmptyWorld.world",
+#            'world': "DefaultEmptyWorld.world",
+            'world': "Sofa.sdf",
```

Comment:
Is this a leftover?

Reply:
This was part of the first commit to easily reproduce and test the issue. I do not expect this to be left forever in the code.

Reply:
Yes but here you edited an existing environment, and now there's a sofa in it :) It cannot go upstream.

The right way to avoid regressions is developing a unit test that fails with the current status and it get fixed in the PR. Can you please amend this edit and add a test instead? You can find other examples [here](https://github.com/robotology/gym-ignition/tree/master/tests/python).